### DISCUSSION TITLE
tini -g ---

### DISCUSSION THREAD

**Discussion by Zoey2936** [nextcloud/all-in-one#4490] [Ideas] [answered]
Mailcow seems to use the tini `-g` option (https://github.com/mailcow/mailcow-dockerized/blob/36b5cccd186090d726de62b6b00d1e842e67aacd/data/Dockerfiles/clamd/Dockerfile#L25), is this something we also need? The only documentation I've found about that is this: https://github.com/krallin/tini#process-group-killing

**Comment by szaimen** [comment]
None of our containers uses tini directly afaics so not sure if this is applicable here?

**Reply by Zoey2936** [reply]
We use docker init which is basically tini, but my question is if this arg would improve tini (if yes we would need to install tini)

**Reply by szaimen** [reply]
I cannot tell honestly and would still like to keep using docker init.

**Reply by Zoey2936** [reply, marked as answer]
then we keep it like it is now
